torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -32,7 +32,7 @@ Fusion* FusionGuard::getCurFusion() {
   return ACTIVE_FUSION;
 }
 
-void swap(Fusion& a, Fusion& b) noexcept {
+TORCH_CUDA_CU_API void swap(Fusion& a, Fusion& b) noexcept {
```

Comment:
Windows compilation fails on this line with:
```
2021-09-16T10:43:07.5518815Z C:\actions-runner\_work\pytorch\pytorch\pytorch-1240968219\torch\csrc\jit\codegen\cuda\fusion.cpp(35): error C2375: 'torch::jit::fuser::cuda::swap': redefinition; different linkage
2021-09-16T10:43:07.5521053Z C:\actions-runner\_work\pytorch\pytorch\pytorch-1240968219\torch/csrc/jit/codegen/cuda/ir_base_nodes.h(63): note: see declaration of 'torch::jit::fuser::cuda::swap'
```

Reply:
Thanks! missed that one in the log :pleading_face: